common/djangoapps/third_party_auth/models.py

```diff
@@ -562,7 +562,7 @@ def get_setting(self, name):
             return getattr(settings, 'SOCIAL_AUTH_SAML_SP_PRIVATE_KEY', '')
         other_config = {
             # These defaults can be overriden by self.other_config_str
-            "EXTRA_DATA": ["attributes"],  # Save all attribute values the IdP sends into the UserSocialAuth table
+            "GET_ALL_EXTRA_DATA": True,  # Save all attribute values the IdP sends into the UserSocialAuth table
```

Comment:
Yeah, there's no migration being done here, so it does look like `UserSocialAuth.extra_data`  will differ between the releases in this respect.

What we can do to keep things consistent is find a way of shoving this stuff back into a dictionary with `attributes` as its key, but I think, moving forward, there's no harm in keeping it this way and a release note about this would be the better, easier, and simpler option.